Mathlib/RingTheory/Nilpotent/Exp.lean

```diff
@@ -36,7 +40,7 @@ algebra, exponential map, nilpotent
 
 namespace IsNilpotent
 
-variable {A : Type*} [Ring A] [Algebra ℚ A]
+variable {A : Type*} [Ring A] [Module ℚ A]
```

Comment:
If there's a concrete example where this generalization is useful, can you mention it in the PR description?

Reply:
It's mathematically equivalent. Indeed the following succeeds:
```lean
example : Algebra ℚ A := Algebra.ofModule smul_mul_assoc mul_smul_comm
```
So it doesn't really matter but I have decided to prefer the superficially typeclass assumption.